api/src/main/java/org/apache/gravitino/authorization/Role.java

```diff
@@ -53,4 +52,13 @@ public interface Role extends Auditable {
    * @return The securable objects of the role.
    */
   List<SecurableObject> securableObjects();
+
+  /**
+   * The securable objects count of the role.
+   *
+   * @return The scecurable objects count of the role.
+   */
+  default int securableObjectsCount() {
```

Comment:
I think it would be better rename to `count()`, what do you think?

Reply:
If we add more fields, it may be confusing for users. For example, if we add child role in the role in the future. If I use `count`, I don't know it's the child role count or objects count.